src/ConfigDefinition.php

```diff
@@ -39,6 +39,7 @@ protected function getParametersDefinition(): ArrayNodeDefinition
                         ->scalarNode('title')->isRequired()->cannotBeEmpty()->end()
                         ->scalarNode('identifier')->end()
                         ->booleanNode('disabled')->end()
+                        ->scalarNode('anchorIdentifier')->end()
```

Comment:
Tomuhle teda moc nerozumím, protože v masteru to je (https://github.com/keboola/gooddata-writer-v3/blob/master/src/ConfigDefinition.php#L42), ale už nemám sílu to zkoumat. ;o)